manim/mobject/functions.py

```diff
@@ -62,6 +66,28 @@ def construct(self):
                 self.add(axes, curve1)
                 self.set_camera_orientation(phi=80 * DEGREES, theta=-60 * DEGREES)
                 self.wait()
+
+    .. attention::
+        If your function has discontinuities, you'll have to specify the location
+        of the discontinuities manually. See the following example for guidance.
+
+    .. manim:: DiscontinuousExample
+        :save_last_frame:
+
+        class DiscontinuousExample(Scene):
+            def construct(self):
+                ax1 = NumberPlane((-3, 3), (-4, 4))
+                ax2 = NumberPlane((-3, 3), (-4, 4))
+                VGroup(ax1, ax2).arrange()
+                discontinuous_function = lambda x: (x ** 2 - 2) / (x ** 2 - 4)
+                before = ax1.plot(discontinuous_function, color=RED)
+                after = ax2.plot(
+                    discontinuous_function,
+                    discontinuities=[-2, 2],  # discontinuous points
+                    dt=0.1,  # left and right tolerance of discontinuity
+                    color=GREEN,
+                )
+                self.add(ax1, ax2, before, after)
```

Comment:
Following @Darylgolden's suggestion:

```suggestion
                without_discontinuities = ax1.plot(discontinuous_function, color=RED)
                with_discontinuities = ax2.plot(
                    discontinuous_function,
                    discontinuities=[-2, 2],  # discontinuous points
                    dt=0.1,  # left and right tolerance of discontinuity
                    color=GREEN,
                )
                self.add(ax1, ax2, without_discontinuities, with_discontinuities)
```

Reply:
I think I still prefer "incorrect" and "correct", without_discontinuities and with_discontinuities is a little confusing (since it's not that the function has no discontinuities, just that the discontinuities argument was not passed)

Reply:
Oops, forgot abt this pr.. I think daryl's original idea of incorrect vs correct might be more fitting, since the function does contain discontinuities in both examples regardless of the implementation, it's a matter of tweaking the parameters imo. what do you think?

Reply:
Feel free to change it. Using "correct" in any connection to Manim's plotting feels a bit like a stretch to me, but ultimately I don't care about the variable names; could just a well be `a` and `b` or `left` and `right` (pun intended).

Reply:
savage

Reply:
Maybe that sounded a bit rougher than I had intended. 😅

Thanks for everyone's effort here!